library/src/scala/quoted/Quotes.scala

```diff
@@ -204,11 +204,22 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
    */
   trait reflectModule { self: reflect.type =>
 
+    extension (expr: Expr[Any]):
+
+      /** Returns the `Tree` representation this expression.
+       *  Same as `asTerm` but typed as a `Tree`.
+       */
+      def asTree: Tree
```

Comment:
It is sufficient. I added this one because it is an intuitive name. Maybe it is better to remove it. Users can always define their extension method that does the same in a couple of lines of they prefer `asTree` in their use cases. It can also be added back in a future version of the library.

Reply:
I think it's better to remove and only keep `asTerm`. For IDE support, the two versions make no difference, which is different from the case for `size` and `length`.

Reply:
I will remove `asTree`